README.md

```diff
@@ -1,10 +1,18 @@
 # Proxless
```

Comment:
comment - docs changes != feat/interfaces.

Reply:
Yeah sorry about that, I got excited with this refactoring and ended up refactoring everything. 😛